recipe/run_conda_forge_build_setup_win.bat

```diff
@@ -16,5 +16,8 @@ conda install -n root --yes --quiet jinja2 conda-build anaconda-client
 conda install -n root --yes --quiet vs2008_express_vc_python_patch
 call setup_x64
 
+:: Set the conda-build working directory to a smaller path
+set "CONDA_BLD_PATH=C:\\bld\\"
```

Comment:
I agree with him that putting it not in `C:\\` is probably a good idea. Unless, of course, we have demonstrated cases where even having `bld` is a problem, which I don't think we have ATM.